app/src/main/java/de/tum/in/tumcampusapp/activities/BarrierFreeInfoActivity.java

```diff
@@ -0,0 +1,51 @@
+package de.tum.in.tumcampusapp.activities;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.ArrayAdapter;
+import android.widget.Button;
+import android.widget.ListAdapter;
+import android.widget.ListView;
+
+import de.tum.in.tumcampusapp.R;
+import de.tum.in.tumcampusapp.activities.generic.BaseActivity;
+
+
+public class BarrierFreeInfoActivity extends BaseActivity {
+
+    private Button button1;
```

Comment:
This button is just used to test the contact person activity. It should be removed.